kolibri/core/assets/src/api_resource.js

```diff
@@ -95,10 +100,14 @@ class Collection {
    * Method to fetch data from the server for this collection.
    * @param {object} extraParams - an object of parameters to be parsed into GET parameters on the
    * fetch.
+   * @param {boolean} force - fetch whether or not it's been synced already.
    * @returns {Promise} - Promise is resolved with Array of Model attributes when the XHR
    * successfully returns, otherwise reject is called with the response object.
    */
-  fetch(extraParams = {}) {
+  fetch(extraParams = {}, force = false) {
+    if (!force && this.synced) {
+      return Promise.resolve(this.models.map((model) => model.attributes));
```

Comment:
Can use `this.data` for this - I defined it as a getter as it's such a common mapping.